.github/workflows/eden_old.yml

```diff
@@ -0,0 +1,160 @@
+---
+name: Eden
+# yamllint disable-line rule:comments, rule:truthy
+on:
+  workflow_dispatch
```

Comment:
Did you check that this workflow works in case of manual `workflow_dispatch` run?
I am not sure that we will pass the check `if: ${{ github.event.review.state == 'approved' || github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags') }}`. It looks like we will be able to test only the head of master.

Reply:
`
if: ${{ github.event.review.state == 'approved' || github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags') }}`

The plan is to test the WF on master. I have added the `workflow_dispatch` only for testing needs and will remove it soon.